src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
+import { User } from '../../types/User';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
-);
+type Props = {
+  user: User | null,
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  if (user) {
```

Comment:
You don't need to check it because on the upper level you already did it
`{post.user && <UserInfo user={post.user} />}`

Reply:
Fix it